docs/examples/keras-to-snn.ipynb

```diff
@@ -282,7 +282,7 @@
     "            plt.ylabel(\"Firing rates (Hz)\")\n",
     "        plt.xlabel(\"Timestep\")\n",
     "        plt.title(\n",
-    "            \"Neural activities (conv0 mean=%dHz max=%dHz)\" % (rates.mean(), rates.max())\n",
+    "            f\"Neural activities (conv0 mean={rates.mean()}Hz max={rates.max()}Hz)\"\n",
```

Comment:
Should have spaces before units (Hz). Also, we were formatting these as integers before; I might recommend one decimal place.